Packs/Cybereason/Integrations/Cybereason/Cybereason_test.py

```diff
@@ -307,3 +314,20 @@ def test_query_file(mocker):
     assert 'Cybereason file query results' in result[0]['HumanReadable']
     assert result[0]['EntryContext']['Cybereason.File(val.MD5 && val.MD5===obj.MD5 || val.SHA1 && '
                                      'val.SHA1===obj.SHA1)'][0]['Machine'] == 'desktop-p0m5vad'
+
+
+def test_malop_processes_command(mocker):
+    mocker.patch.object(demisto, 'params', return_value=params)
+    mocker.patch.object(demisto, 'args', return_value={"malopGuids": ["11.-6236127207710541535"]})
+    raw_response = util_load_json('test_files/malop_processes_raw_response.json')
+    mocker.patch('Cybereason.malop_processes', return_value=raw_response)
+    mocker.patch.object(demisto, 'results')
+    import Cybereason
```

Comment:
move this to the top of the test please.
And you can do:
From Cybereason import malop_processes_command
And then use the command without the prefix